cmd/minishift/cmd/start.go

```diff
@@ -324,7 +324,7 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
 	if !isRestart {
 		fmt.Println("-- Minishift VM will be configured with ...")
-		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   Memory:   ", units.BytesSize(float64(machineConfig.Memory*units.MiB)))
```

Comment:
* Docker machine used MB
```
  --kvm-memory "1024"                                                                                  Size of memory for host in MB
```
  and doesn't even come close to showing the memory assignment, except for `docker-machine inspect` to dump the `config.json`

  * minikube uses memory from `minikube config set memory [size]` in MB.